group-generators/helpers/data-providers/dune/interface-schema.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
Same here:

In order to use the DataProvider in the [Factory](https://factory.sismo.io/), you need to import it into the DataProvider index (group-generators/helpers/data-providers/index.ts)

- Import the `duneInterfaceSchema`
- Add the `duneInterfaceSchema` to the `dataProvidersInterfacesSchemas` object
- Add your count function to the `dataProvidersAPIEndpoints`
```
DuneProvider: {
  executeQueryCount: async (_: any) =>
  new DuneProvider().executeQueryCount(_),
},
...
```

Reply:
Thanks! 
How can we insert the API key when using the factory? The Dune query won't run without it.
Could we add it as an argument for the factory users?

Reply:
Hey, we need to add it in the "infra" part like for the GITHUB_TOKEN
for instance you have it here in the workflows https://github.com/sismo-core/sismo-hub/blob/main/.github/workflows/deploy_prod.yml#L47 
and then it goes into the github action, then the serverless yaml https://github.com/sismo-core/sismo-hub/blob/main/serverless.yaml 
and env vars management into the sismo-hub :-) 

We can handle this part if you prefer

Reply:
Thanks, I can update the workflows folder. 
Looks like there's 2 naming conventions, 
`github-token: ${{ secrets.SH_GITHUB_TOKEN }}`
and
`transpose-api-key: ${{ secrets.TRANSPOSE_API_KEY }}`

Do you want the `SH_` before the key?

Reply:
`DUNE_API_KEY` is better. the `SH_GITHUB_TOKEN` is due to a naming conflict in the github action auto injected GITHUB_TOKEN